GTMAppAuth/Sources/Public/GTMAppAuth/GTMAppAuthFetcherAuthorization.h

```diff
@@ -71,8 +71,11 @@ typedef void (^GTMAppAuthFetcherAuthorizationCompletion)(NSError *_Nullable erro
         library.
     @discussion Enables you to use AppAuth with the GTM Session Fetcher library.
  */
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
@paulb777 note that ignoring "-Wdeprecated-implementations" does not seem to be needed to silence warnings in the implementation.  Did you see something different with firebase/firebase-ios-sdk#10131?

Reply:
Yes, I needed to add `-Wdeprecated-implementations` around the implementation of the deprecated API to eliminate the second warning.

Reply:
Ah, yes.  Now I see the implementation warning when building with CocoaPods.  Updated to ignore.